packages/IMAPClient-Tests.package/ICEndPointCommandTest.class/instance/setUp.st

```diff
@@ -18,4 +18,4 @@ setUp
 	self endPoint stream connected: true.
 	"self endPoint stream server responses: self endPoint stream server selectedResponses."
 	"Transcript show: (String cr, 'last response:', String cr, (self endPoint stream server selectedResponses at: '. FETCH 45 (body[HEADER.FIELDS (from to subject date)])') , String cr, String cr)."
-	self endPoint lastResponse: (self endPoint stream server selectedResponses at: '. FETCH 45 (body[HEADER.FIELDS (from to subject date)])').
\ No newline at end of file
+	self endPoint lastResponse: (self endPoint stream server selectedResponses at: '. FETCH 45 (body[HEADER.FIELDS (from to subject date)])').
```

Comment:
Würde hier auch wieder den Punkt löschen.. generell in allen Methoden am Ende :).